package.json

```diff
@@ -14,5 +14,7 @@
     "author": {
         "name": "Team Zomsa",
         "url": "https://github.com/team-zomsa"
-    }
+    },
+    "dependencies": {},
+    "testables": ["com.team-zomsa.aplib-unity"]
```

Comment:
This should be removed here as it won't do anything, testables only works for user's own Packages folder manifest.

Reply:
You're right